graphql_api/types/account/account.py

```diff
@@ -29,3 +37,20 @@ def resolve_total_seat_count(account: Account, info: GraphQLResolveInfo) -> int:
 @sync_to_async
 def resolve_activated_user_count(account: Account, info: GraphQLResolveInfo) -> int:
     return account.activated_user_count
+
+
+@account_bindable.field("organizations")
+@convert_kwargs_to_snake_case
```

Comment:
we no longer need this decorator since we have this now! https://github.com/codecov/codecov-api/blob/9aa6458ad023fe0ec55993ad9134bbd0af5fb3f7/graphql_api/schema.py#L7